src/honey_badger.rs

```diff
@@ -97,7 +97,7 @@ pub struct HoneyBadger<C, NodeUid: Rand> {
     /// The messages that need to be sent to other nodes.
     messages: MessageQueue<NodeUid>,
     /// The outputs from completed epochs.
-    output: VecDeque<Batch<C, NodeUid>>,
+    pub(crate) output: Vec<Batch<C, NodeUid>>,
```

Comment:
Why does that have to be `pub(crate)`? Shouldn't everything outside this module just use the `Step`?

Reply:
I think `output` gets accessed directly at the time when `HoneyBadger` is restarted.